src/index.js

```diff
@@ -1,5 +1,21 @@
-import './style.scss';
-import data from './data.js';
+import { getScores, postScore } from './api.js';
+import startDark from './darkmode.js';
 import showData from './display.js';
+import './style.scss';
+
+startDark();
+
+document.addEventListener('submit', (e) => {
+  const [name, score, errors] = ['nameInput', 'scoreInput', 'errors'].map((id) => document.getElementById(id));
+  if (score.value > 0) {
+    postScore({ user: name.value, score: score.value }).then((data) => data.message);
+    document.forms[0].reset();
+  } else {
+    errors.innerHTML = '* Please, enter new score';
+  }
+  e.preventDefault();
+});
```

Comment:
[OPTIONAL] You've done a wonderful job, congratulations! I think maybe you could work on ordering the list or giving the user the possibility to do it. Or refreshing automatically after a score is submitted. I just wanted to give you a couple of ideas that you might like to implement if you have the time!

Cheers and Happy coding! :rocket:

Reply:
@D4nielJ Thanks for your suggestions :100: i will work on it ASAP :+1: